src/Vendoring/OpenTelemetry.Instrumentation.EventCounters/EventCountersInstrumentationOptions.cs

```diff
@@ -0,0 +1,59 @@
+// <copyright file="EventCountersInstrumentationOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
```

Comment:
Where is this coming from? Might be an older version? (the latest version is https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/41c0a2d310c1dbbb880fe35641f524d6ba1bf3a6/src/OpenTelemetry.Instrumentation.EventCounters/EventCountersInstrumentationOptions.cs#L1-L2)

Reply:
tags/Instrumentation.EventCounters-1.5.1-alpha.1 which is the last release